docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+
+The two primary package-based target frameworks used with .NET Core are:
+
+- `netstandard`
+- `netcoreapp`
+
+.NET Standard
+-------------
+
+The .NET Standard (TFM: `netstandard`) framework represents the APIs defined by and built on top of the [.NET Standard Library](../concepts/dotnet-standard-library.md). Libraries that are intended to run on multiple runtimes should use this target. They will be supported on any .NET Standard compliant runtime, such as .NET Core, .NET Framework and Mono/Xamarin. Each of these runtimes supports a set of .NET Standard versions, depending on which APIs they implement. 
+
+The `NETStandard.Library` metapackage targets the `netstandard` framework. The most common way to target `netstandard` is by referencing this metapackage. It describes and provides access to the ~40 .NET  libraries and associated APIs that define the .NET Standard Library. You can reference additional packages that target `netstandard` to get access to additional APIs. These higher-level packages can be thought of as "targeting netstandard" or as "a .NET Standard Library" but they do not define "the .NET Standard Library".
+
+A given [NETStandard.Library version](versioning.md) matches the highest `netstandard` version it exposed (via its closure). The framework reference in project.json is used to select the correct assets from the underlying packages. In this case, `netstandard1.5` assets are required, as opposed to `netstandard1.4` or `net46`, for example. 
+
+```json
+{
+  "dependencies": {
+    "NETStandard.Library": "1.5.0"
+  },
+  "frameworks": {
+    "netstandard1.5": {}
+  }
+}
+```
+
+The framework and metapackage references in project.json do not need to match. For example, the following project.json is valid.
+
+```json
+{
+  "dependencies": {
+    "NETStandard.Library": "1.5.0"
+  },
+  "frameworks": {
+    "netstandard1.3": {}
+  }
+}
+```
+
+It may seem strange to target `netstandard1.3` but use the 1.5.0 version of `NETStandard.Library`. It is a valid use-case, since the metapackage maintains support for older `netstandard` versions. It could be the case you've standardized on the 1.5.0 version of the metapackage and use it for all your libraries, which target a variety of `netstandard` versions. With this approach, you only need to restore `NETStandard.Library` 1.5.0 and not earlier versions. The reverse would not be valid. You cannot use `netstandard1.5` with the 1.3.0 version of `NETStandard.Library` since it will not reference packages with `netstandard1.5` assets.
```

Comment:
Good point! No, we are not planning on releasing earlier versions. It's not necessary for any scenario. I'll update the text.